client/reader/sidebar/sidebarHOC.jsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getSidebarIsCollapsed } from 'calypso/state/ui/selectors';
+
+export const sidebarHOC = ( Component ) => {
+	return ( props ) => {
+		const reduxDispatch = useDispatch();
+		const sidebarIsCollapsed = useSelector( getSidebarIsCollapsed );
+
+		useEffect( () => {
+			if ( sidebarIsCollapsed ) {
+				reduxDispatch( recordTracksEvent( 'calypso_toggle_sidebar' ) );
+				reduxDispatch( savePreference( 'sidebarCollapsed', ! sidebarIsCollapsed ) );
```

Comment:
It feels a bit invasive to change the collapsed/expand sidebar preference of the user.

How about if we just always display expanded version for the Reader, and disregard the user's preference in that case?

Reply:
Hmm, UX wise also it will be better to disregard the user's preference & not to alter its choice. Will do changes for it.